src/domdiv/card_db/xx/cards_xx.json

```diff
@@ -1,6 +1,6 @@
 {
     "Alms": {
-        "description": "Once per turn: If you have no Treasures in play, gain a card costing up to 4 Coin.",
+        "description": "One per turn: If you have no Treasures in play, gain a card costing up to 4 Coin.",
```

Comment:
This change seems wrong and should be reverted.

Reply:
Yes, I'm just testing that it's auto-committing the processed version of the card_db properly when the developer did not.